PA_20.py

```diff
@@ -0,0 +1,86 @@
+#LC 198 - House Robber
+#Recursive Solution
+#Time Complexity - Exponential (what will be the exact? 2^n?)
+#Space Complexity - O(n) Again, doubt. Could you clear it out?

```

Comment:
O(N) should be correct, recursion is called on all the element either in the chosen case or not chosen case
The recursive stack will have address of at most N/2 which is basically O(N)